coordinator/core/marbleapi.go

```diff
@@ -37,17 +37,17 @@ type reservedSecrets struct {
 	MarbleCert manifest.Secret
 }
 
-// Defines the "MarbleRun" prefix when mentioned in a manifest
+// Defines the "MarbleRun" prefix when mentioned in a manifest.
 type secretsWrapper struct {
 	MarbleRun reservedSecrets
 	Secrets   map[string]manifest.Secret
 }
 
-// Activate implements the MarbleAPI function to authenticate a marble (implements the MarbleServer interface)
+// Activate implements the MarbleAPI function to authenticate a marble (implements the MarbleServer interface).
 //
 // Verifies the marble's integrity and subsequently provides the marble with a certificate for authentication and application-specific parameters as defined in the Coordinator's manifest.
 //
-// req needs to contain a MarbleType present in the Coordinator's manifest and a CSR with the Subject and DNSNames set with desired values.
+// Req needs to contain a MarbleType present in the Coordinator's manifest and a CSR with the Subject and DNSNames set with desired values.
```

Comment:
```suggestion
// req needs to contain a MarbleType present in the Coordinator's manifest and a CSR with the Subject and DNSNames set with desired values.
```
`req` refers to the variable, so should not be uppercase